code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2446,7 +2446,7 @@
 /datum/reagent/pax
 	name = "Pax"
 	description = "A colorless liquid that suppresses violence in its subjects."
-	color = "#AAAAAA55"
+	color = "#aaaaaaff"
```

Comment:
```suggestion
	color = rgb(170, 170, 170)
```

just to make it a bit more readable, I suppose.

`ff` is 255, the maximum alpha, anyways